etc/kayobe/kolla/globals.yml

```diff
@@ -22,26 +22,26 @@ monasca_agent_authorized_roles:
 # Container image tags.
 # These should be allowed to change independently, as services are updated.
 
-barbican_tag: 6.1.0.2-1
-cron_tag: 6.1.0.2-1
-fluentd_tag: 6.1.0.2-1
-glance_tag: 6.1.0.2-1
-haproxy_tag: 6.1.0.2-1
-heat_tag: 6.1.0.2-1
-horizon_tag: 6.1.0.2-1
-ironic_tag: 6.1.0.2-1
-iscsid_tag: 6.1.0.2-1
-keepalived_tag: 6.1.0.2-1
-keystone_tag: 6.1.0.2-1
-kolla_toolbox_tag: 6.1.0.2-1
-magnum_tag: 6.1.0.2-2
-manila_tag: 6.1.0.2-1
-mariadb_tag: 6.1.0.2-1
-memcached_tag: 6.1.0.2-1
-neutron_tag: 6.1.0.2-2
-nova_tag: 6.1.0.2-1
-openvswitch_tag: 6.1.0.2-1
-rabbitmq_tag: 6.1.0.2-1
+barbican_tag: 6.2.1.1-1
```

Comment:
Is it best practice to pin all of these then?

Reply:
Yes, since you can then bump `kayobe_openstack_release` without affecting existing services.

Reply:
Perhaps we should rebase on top of this which has been applied?

https://github.com/SKA-ScienceDataProcessor/alaska-kayobe-config/pull/136

Fluentd looks like it will be downgraded, but I think that should be ok.

Reply:
I can skip the common tag if that helps?